src/context/AppContext.js

```diff
@@ -305,15 +305,18 @@ export const AppProvider = (props) => {
   async function loadOrganizations() {
     const orgs = await api.getOrganizations();
     const { stakeholders } = await stakeholder_api.getStakeholders();
-    const results = stakeholders.map((s) => {
-      return {
-        id: s.id,
-        stakeholder_uuid: s.id,
-        name: s.org_name || s.first_name + ' ' + s.last_name,
-        type: s.type,
-      };
+    const results = [];
+    stakeholders.forEach((s) => {
```

Comment:
This isn't populating `results`

Reply:
@gwynndp Can you double check this method when you get a chance?
It appears to duplicate the orgs in the org list (assuming Stakeholders API has the same data).

Reply:
I adjusted it to only show the stakeholders that the user is allowed to see.    It may mean that temporarily we won't see Person type stakeholders in the list or any legacy orgs that don't have a corresponding stakeholder  with the same uuid.  (Right now there is only one, a duplicate "test2" org)